test/amchartFactory.js

```diff
@@ -82,10 +83,8 @@
                     ;
                     col.xAxis(0).axisType("ordinal");
                     col.yAxis(0).axisType("linear");
-                    col.yAxis(1)
-                        .axisType("linear")
-                        .position("right")
-                    ;
+                    col.yAxis(1).axisType("linear");
+                    col.yAxis(1).position("right")  ; 
```

Comment:
u asked me to put it back ... so i did ... it will work either way lets just leave it ... it shows u can do it both ways